packages/terra-demographics-banner/src/DemographicsBanner.module.scss

```diff
@@ -106,5 +109,11 @@
     .identifiers {
       text-align: right;
     }
+
+    .row {
```

Comment:
Since this is used specifically for the second row containing patient identifiers, could this be named something like patient-identifier-row?

Also never knew about these CSS properties but this is sweet, also wraps nicely when there is overflow
![image](https://github.com/cerner/terra-core/assets/37750902/2c4de0d7-b7bd-4c1e-991f-f0f7325e675c)

Reply:
Changed to 'identifier-row': https://github.com/cerner/terra-core/pull/3796/commits/f0bd4ce0afcf337a5da63963902f5226070c034a